cli/js/index.js

```diff
@@ -26,8 +26,8 @@ async function showStats(node) {
     return;
   }
 
-  const squareRows = networkHead.dah.row_roots.length;
-  const squareCols = networkHead.dah.column_roots.length;
+  const squareRows = networkHead.dah.row_roots().length;
+  const squareCols = networkHead.dah.column_roots().length;
```

Comment:
I'm not sure if I like this breaking change because JS it doesn't give even a runtime error if user does not migrate.